core/src/main/java/bio/terra/pearl/core/service/search/terms/functions/LowerFunction.java

```diff
@@ -0,0 +1,66 @@
+package bio.terra.pearl.core.service.search.terms.functions;
+
+import bio.terra.pearl.core.model.search.SearchValueTypeDefinition;
+import bio.terra.pearl.core.service.search.EnrolleeSearchContext;
+import bio.terra.pearl.core.service.search.sql.EnrolleeSearchQueryBuilder;
+import bio.terra.pearl.core.service.search.terms.SearchTerm;
+import bio.terra.pearl.core.service.search.terms.SearchValue;
+import org.jooq.Condition;
+
+import java.util.List;
+import java.util.Optional;
+
```

Comment:
We need to have some docs here -- either an example or a link to a zendesk page

Reply:
I'm working on zendesk docs right now! I'll also add an example but we will have that to go from as well.